File: bookshop-front-angular/src/models/Book.ts
export class Book {
    public id: number = 0;
    public isbn_13: string = "";
    public publicId: string = "";
    public author: string = "";
    public price: number = 0;
    public title: string = "";
    public subTitle?: string = "";

    constructor(object: any) {
        this.id = object.id;
        this.isbn_13 = object.isbN_13;
        this.publicId = object.publicId;
        this.author = object.author;
        this.price = object.price;
        this.title = object.title;
        this.subTitle = object.subTitle;
    }
}

File: bookshop-back-api/bookshop.webapi/Models/Book.cs
using bookshop.webapi.Models.CartFolder;
using bookshop.webapi.Models.OrderFolder;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace bookshop.webapi.Models
{
    public class Book
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }
        public string ISBN_13 { get; set; }
        public string PublicId { get; set; }
        public string Author { get; set; }
        public float Price { get; set; }
        public string Title { get; set; }
        public string SubTitle { get; set; }
        public List<Comment> Comments { get; set; }
        public List<CartItem> CartItems { get; set; }
        public List<OrderItem> OrderItems { get; set; }
    }
}